package.json

```diff
@@ -4,9 +4,10 @@
     "@types/material-ui": "^0.21.7",
     "@types/react-dom": "^16.9.0",
     "@types/react-router-dom": "^5.0.1",
+    "@types/react": "^17.0.58",
```

Comment:
Should this be `"~17.0.58"`? I think these types libraries guarantee compatibility within the minor version?